README.md

```diff
@@ -9,85 +9,117 @@ website: cltl.nl
-    cd StoryTeller
-    chmod +wrx install.sh
-    ./install.sh
+git clone https://github.com/cltl/StoryTeller
+cd StoryTeller
+chmod +wrx install.sh
+./install.sh
 ```
 
-The install.sh will build the binary through apache-maven-2.2.1 and the pom.xml and move it to the "lib" folder.
+The install.sh will build the binary through apache-maven-2.2.1 and the pom.xml
+and move it to the "lib" folder.
 
 # LICENSE
-    StoryTeller is free software: you can redistribute it and/or modify
-    it under the terms of the The Apache License, Version 2.0:
-        http://www.apache.org/licenses/LICENSE-2.0.txt.
-
-    StoryTeller is distributed in the hope that it will be useful,
-    but WITHOUT ANY WARRANTY; without even the implied warranty of
-    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-    License for more details.
 
+```
+StoryTeller is free software: you can redistribute it and/or modify
+it under the terms of the The Apache License, Version 2.0:
+    http://www.apache.org/licenses/LICENSE-2.0.txt.
+
+StoryTeller is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+License for more details.
+```
 
 # DESCRIPTION of the CLTL/StoryTeller component
 
-Toolkit to query the NewsReader KnowledgeStore with SPARQL and create a JSON story.
-The NewsReader KnowledgeStore contains event-centric knowledge graphs (ECKGs) according to the
-Simple Event Model (SEM, Verhage et al 2011) and the Grounded Annotation and Source Perspective model
-(GRaSP, Vossen et al 2016). ECKGs represent instances of events with their participants and their anchoring in time.
-Instance representations of events and participants is done using URIs and a whole range of properties is expressed for these
-instances, among which: the labels used to make reference, semantic types of which they are instances,
-the offsets where they are mentioned in the text, the semantic roles between participants and events,
-the topic of the document in which the event is mentioned, etc.
-The event data can be queried through the instance level (participants), the phrase level and the type level,
-where you can combine constraints on the events and the participants.
-
-For each event mention, there is a representation of the source to which the mentioning of the event is attributed and
-the perspective of the source to the event. The source can be the author or somebody cited. The perspective reflects
-the sentiment, the denial/confirmation, the certainty, the time perspective. Likewise you can query for events
-mentioned by specific sources and/or on which a certain perspective is expressed.
-
-The JSON story that is returned for a query consists of a list of ECKGs, where each ECKG has the following structure (see also Zwaan et al 2016):
+Toolkit to query the NewsReader KnowledgeStore with SPARQL and create a JSON
```

Comment:
a PR to the original repo was always the plan, but only after it is complete in terms of documentation and testing.